namer/namer.cc

```diff
@@ -1308,6 +1352,9 @@ class SymbolDefiner {
                 case DefinitionKind::Class:
                     modifyClass(ctx.withOwner(owner), modifier);
                     break;
+                case DefinitionKind::Constant:
+                    modifyConstant(ctx.withOwner(owner), modifier);
+                    break;
                 default:
                     Exception::raise("Found invalid modifier");
```

Comment:
Given the suggestion above of changing to `Modifier::Kind` will let us delete this `default` case so that clang can check for exhaustiveness here.

Reply:
Bump on this. If you use `enum class`, you should be able to delete the `default` case and clang will check  you for exhaustiveness.